v8py/v8py.cpp

```diff
@@ -102,6 +117,7 @@ static PyMethodDef v8_methods[] = {
 };
 
 PyObject *null_object = NULL;
+uint16_t *string_buffer = NULL;
```

Comment:
I think I'd prefer this to be statically allocated, like `uint16_t string_buffer[STRING_BUFFER_SIZE]`